ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -544,18 +599,21 @@ export class PrepareMigrationComponent implements OnInit {
     })
   }
 
+
   migrate() {
     this.resetValues()
     let payload: IMigrationDetails = {
       TargetDetails: this.targetDetails,
       DataflowConfig: this.dataflowConfig,
       IsSharded: this.isSharded,
+      DataprocConfig: this.dataprocConfig,
       MigrationType: this.selectedMigrationType,
       MigrationMode: this.selectedMigrationMode,
       skipForeignKeys: this.isForeignKeySkipped
     }
     this.fetch.migrate(payload).subscribe({
       next: () => {
+        //TODO: eenclona@ will update this with migration status of dataproc job
```

Comment:
Is this comment still aplicable?

Reply:
It should be, as i didn't do any new change w.r.t. migration status of dataproc jobs, I'll check with eenclona@ once

Reply:
This comment is no longer applicable and can be discarded.

Reply:
Comment removed
Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/519/commits/c8de48aca03cdf67292e4652162a005987a6f0f2